local/test-binary

```diff
@@ -90,15 +90,19 @@ if [[ "${ANALYZER_OPTS}" == *WITH_VALGRIND=ON* ]]; then
     PARALLEL=$((PARALLEL/3))
   fi
 fi
-#
+
+if [[ "${ANALYZER_OPTS}" == *WITH_ASAN=ON* ]] || [[ "${ANALYZER_OPTS}" == *WITH_UBSAN=ON* ]]; then
```

Comment:
`${ANALYZER_OPTS}` can also be `-DWITH_MSAN=ON` (even though we do not test this currently).
I am not also sure about `WITH_ASAN_SCOPE=ON: as far as I remember `WITH_ASAN_SCOPE=ON` always comes along with `WITH_ASAN=ON` - please double check.

Reply:
you are right, WITH_ASAN_SCOPE should be used with WITH_ASAN

from documentation:
-DWITH_ASAN_SCOPE=bool
Whether to enable the AddressSanitizer -fsanitize-address-use-after-scope Clang flag for use-after-scope detection. The default is off. To use this option, -DWITH_ASAN must also be enabled.